install_proto.sh

```diff
@@ -1,9 +1,9 @@
 JAVA_LIB_VERSION=1.29.0
 URL=https://repo1.maven.org/maven2/io/grpc/protoc-gen-grpc-java/$JAVA_LIB_VERSION/protoc-gen-grpc-java-$JAVA_LIB_VERSION-linux-x86_64.exe
-wget $URL -O protoc-gen-grpc-java
-chmod +x protoc-gen-grpc-java
+sudo wget $URL -O /usr/bin/protoc-gen-grpc-java
+sudo chmod +x /usr/bin/protoc-gen-grpc-java
 
 PROTOC_VERSION=3.11.4
```

Comment:
Welches Tool sagt dir, dass diese version outdated ist?

Reply:
we should use docker in favor of this!